core/src/test/java/com/amplifyframework/logging/AndroidLoggingPluginTest.java

```diff
@@ -131,6 +137,39 @@ public void noContentLoggedAtThresholdNone() {
         assertTrue(systemLog.getLines().isEmpty());
     }
 
+    /**
+     * When the application is not in a debuggable build, logs are not stored.
+     */
+    @Test
+    public void noLogsStored() {
+        Context mockContext = mock(Context.class);
+        ApplicationInfo mockAppInfo = mock(ApplicationInfo.class);
+        mockAppInfo.flags = 0;
+        when(mockContext.getApplicationInfo()).thenReturn(mockAppInfo);
+        AndroidLoggingPlugin plugin = new AndroidLoggingPlugin();
+        plugin.configure(new JSONObject(), mockContext);
+        Logger logger = plugin.forNamespace("logging-test");
+        logger.info("Info log");
+        assertNull(plugin.getLogs());
+    }
+
+    /**
+     * When the application is in a debuggable build, logs are stored.
+     */
+    @Test
+    public void logsAreStored() {
+        Context mockContext = mock(Context.class);
+        ApplicationInfo mockAppInfo = mock(ApplicationInfo.class);
+        mockAppInfo.flags = ApplicationInfo.FLAG_DEBUGGABLE;
+        when(mockContext.getApplicationInfo()).thenReturn(mockAppInfo);
+        AndroidLoggingPlugin plugin = new AndroidLoggingPlugin();
+        plugin.configure(new JSONObject(), mockContext);
+        Logger logger = plugin.forNamespace("logging-test");
+        logger.info("Info log");
+        assertNotNull(plugin.getLogs());
+        assertEquals(plugin.getLogs().size(), 1);
```

Comment:
Nit: It would be a stronger test to assert that the content of the log matches what you expect after the size assertion